jobs/expired-limited-restoration/file_expired_limited_restoration.py

```diff
@@ -0,0 +1,208 @@
+
+    return response.json().get('identifiers', [])
+
+
+def create_put_back_off_filing(app: Flask, identifier: str):
+    """Create a putBackOff filing for the business."""
+    timeout = int(app.config.get('ACCOUNT_SVC_TIMEOUT'))
+    token = get_bearer_token(app, timeout)
+    filing_data = {
+        'filing': {
+            'header': {
+                'date': datetime.utcnow().date().isoformat(),
+                'name': 'putBackOff',
+                'certifiedBy': 'system'
+            },
+            'business': {
+                'identifier': identifier
+            },
+            'putBackOff': {
+                'details': 'Put back off filing due to expired limited restoration.'
+            }
+        }
+    }
+
+    response = requests.post(
+        f'{app.config["LEGAL_API_URL"]}/businesses/{identifier}/filings',
+        json=filing_data,
+        headers={
+            'Content-Type': 'application/json',
+            'Authorization': f'Bearer {token}',
+            'hide-in-ledger': 'true'  # Add this header to hide from ledger
+        },
+        timeout=timeout
+    )
+
+    if not response or response.status_code != 201:
+        app.logger.error(f'Failed to create filing from legal-api. \
+            {response} {response.json()} {response.status_code}')
+        raise Exception  # pylint: disable=broad-exception-raised;
+
+    return response.json()
+
+
+async def run(loop, application: Flask):  # pylint: disable=redefined-outer-name
+    """Run the methods for processing expired limited restorations."""
+    queue_service = ServiceWorker(
+        loop=loop,
+        nats_connection_options=default_nats_options,
+        stan_connection_options=default_stan_options,
+        config=config.get_named_config('production')
+    )
+
+    await queue_service.connect()
+
+    with application.app_context():
+        try:
+            # 1. get businesses that need to be processed
+            businesses = get_businesses_to_process(application)
+
+            if not businesses:
+                application.logger.debug('No businesses to process')
+                return
+
+            application.logger.debug(f'Processing {len(businesses)} businesses')
+
+            # 2. create put back off filing for each business
+            for identifier in businesses:
+                try:
+                    # create putBackOff filing via API
+                    filing = create_put_back_off_filing(application, identifier)
+                    filing_id = filing['filing']['header']['filingId']
+
+                    # queue the filing for processing
+                    msg = {'filing': {'id': filing_id}}
+                    await queue_service.publish(subject, msg)
+                    application.logger.debug(
+                        f'Successfully queued put back off for {identifier}, '
+                        f'put filing {filing_id} on the queue.'
+                    )
```

Comment:
This is not required. It will be processed through the payment flow (legal-api -> pay-api -> business-pay -> entity-filer).

You can remove all the dependencies related to nats queue